lomap/graphgen.py

```diff
@@ -996,10 +996,10 @@ def write_graph(self, output_no_images, output_no_graph):
 
         log = 'The following files have been generated:'
         if not output_no_graph:
-            log += f'\n{self.dbase.options.name}.dot\tGraph file'
+            log += f'\n{self.dbase.options["name"]}.dot\tGraph file'
```

Comment:
should it not be ['name'] over ["name"] to conform to style?

Reply:
it's already inside a f-string, so you'd have to swap both, but not really a big deal either way

Reply:
ah, missed that, check